### PR TITLE
Restore previously deleted CameraView and Notification

### PR DESCRIPTION
**Pull request by patrykka**
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Review by safir-srbd — APPROVED** [review]
@mihashco Thank you for restoring the pages. I believe these are the files that were mistakenly deleted in #1522 that are being restored. I have checked the PR and although there are some minor grammatical changes that need to be made, considering the importance of restoring these pages back to Tizen Docs, they will be made by our team later on. For now, I am approving this PR for merging.

@asm-foysal @mijin85cho I am approving this PR for merging.

**Comment by safir-srbd** [comment]
@patrykka From viewing #1522, I can see that CameraView, CanvasView, and Notification were removed from the overview file `docs/application/dotnet/guides/user-interface/nui/ui-components.md`. Shouldn't these be added back to overview.

**Comment by patrykka** [comment]
@safir-srbd please find applied changes

File: docs/application/dotnet/guides/user-interface/nui/ui-components.md [hunk 1]
```diff
@@ -15,12 +15,15 @@ The following table lists the available UI components:
...
 | [AlertDialog](./nui-components/AlertDialog.md)  | AlertDialog class shows a dialog with title, message, and action buttons. <br>[Guide ](./nui-components/AlertDialog.md) / [API](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.Components.AlertDialog.html)  |  <img src="./nui-components/media/AlertDialog.png" width="600"> <br>[C# code for this page](./codeExamples/AlertDialog.cs) / [XAML page](./codeExamples/AlertDialog.xaml)                |
 | [Button](./nui-components/Button.md) | Button is a common component and describes what action occurs when you select it. A Button can either contain a text or an icon. <br>[Guide ](./nui-components/Button.md) / [API](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.Components.Button.html) | <img src="./nui-components/media/Button.png" width="300"> <br>[C# code for this page](codeExamples/Button.cs) / [XAML page](codeExamples/Button.xaml)               |
+| [CanvasView](./vectorgraphics/CanvasView.md) | A CanvasView is a class for displaying vector primitives. | `CanvasViewView` |
+| [CameraView](./nui-components/CameraView.md) | A camera view that controls and displays camera playback. | `CameraView` |
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/user-interface/nui/ui-components.md:19]: @patrykka Thank you for making the requested additions, but the appearance section isn't like the other rows, as its missing an image, link to C# code, and link to XAML page.

**Review by safir-srbd — COMMENTED** [review]
@patrykka Thank you for making the requested additions. I have left comments regarding the changes that you have made. Please check them.

File: docs/application/dotnet/guides/user-interface/nui/ui-components.md [hunk 2]
```diff
@@ -15,12 +15,15 @@ The following table lists the available UI components:
...
 | [Menu](./nui-components/Menu.md)        | Menu is a class which contains a set of MenuItems and has one of them selected.<br>[Guide ](./nui-components/Menu.md) / [API](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.Components.Menu.html) | <img src="./nui-components/media/Menu.png" width="300"> <br>[C# code for this page](./codeExamples/Menu.cs) / [XAML page](./codeExamples/MenuTestPage.xaml) <br> [C# code Menu definition](./codeExamples/MenuTestMenu.xaml.cs) / [XAML Menu definition](./codeExamples/MenuTestMenu.xaml) |
+| [Notification](./nui-components/Notification.md) | A notification is used to pop-up a notification window with a content view. |<br>[Guide ](./nui-components/Notification.md) / [API](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.Components.Notification.html)| `Notification` |
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/user-interface/nui/ui-components.md:26]:
```suggestion
| [Notification](./nui-components/Notification.md) | A notification is used to pop-up a notification window with a content view. <br>[Guide ](./nui-components/Notification.md) / [API](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.Components.Notification.html)| `Notification` |
```

**Review by safir-srbd — COMMENTED** [review]
@patrykka Thank you for making the suggested changes. After reviewing everything, I found one minor issue in the new changes you made. Please make the change at your earliest convenience.

**Review by safir-srbd — APPROVED** [review]
@patrykka Thank you for making the suggested changes. All seems to be fine for now.

@mijin85cho @asm-foysal I am approving this PR for merging.

**Pull request merged**